src/lints/struct_added_without_non_exhaustive.ron

```diff
@@ -0,0 +1,57 @@
+SemverQuery(
+    id: "struct_added_without_non_exhaustive",
+    human_readable_name: "pub struct added without #[non_exhaustive]",
+    description: "A new public struct was added without #[non_exhaustive], making it harder to evolve in the future.",
+    required_update: Minor,
```

Comment:
Since we don't have any prior "API evolution" advisory lints like this one (this is the first!), we haven't thought through what `required_update` should be here.

Right now, the only allowed options are `Major` and `Minor`. But neither seems quite right. Selecting `Minor` means that this lint won't run on major _or_ minor releases, so new API additions like this will only be linted in patch releases. This doesn't seem quite right =/

Reply:
hmm I see `Minor` is not right I believe, should we put it on-hold till we figure it out